src/server/src/main/java/io/cassandrareaper/jmx/ClusterFacade.java

```diff
@@ -536,7 +536,7 @@ public boolean nodeIsAccessibleThroughJmx(String nodeDc, String node) {
                 Optional.of(node.getHostname()),
                 "org.apache.cassandra.metrics",
                 "ClientRequest",
-                DateTime.now().minusMinutes(1).getMillis()));
+                DateTime.now().minusMinutes(11).getMillis()));
```

Comment:
`11` comes across as a magic number here.
Can you explain why 11 minutes, eg the relationship to the schema (or whatever)

Reply:
The table is partitioned in chunks of 10 minutes. This allows to include the previous partition in the call if the current one has no metric yet.

Reply:
can you put that in as a comment to the code. or better yet, put the number into constant that has a name that explains/hints

Reply:
sure thing